content/id/docs/concepts/services-networking/service.md

```diff
@@ -0,0 +1,1065 @@
+tetap membutuhkan mekanisme _self-registration_ dan _adapter service discovery_ 
+lain yang dapat digunakan berdasarkan API ini.
+
+Untuk _Service_ _"headless"_ alokasi _cluster IP_ tidak dilakukan dan _kube-proxy_ 
+tidak me-_manage_ _Service-Service_, serta tidak terdapat mekanisme _load balancing_ 
+yang dilakukan. Bagaimana konfigurasi otomatis bagi DNS dilakukan bergantung pada 
+apakah _Service_ tersebut memiliki _selector_ yang dispesifikasikan.
+
+### Dengan _selector_
+
+Untuk _Service_ _"headless"_ dengan _selector_, kontroler _Endpoints_ akan membuat suatu 
+_record_ _Endpoints_ di API, serta melakukan modifikasi konfigurasi DNS untuk mengembalikan 
+_A records (alamat)_ yang merujuk secara langsung pada _Pod_ _backend_.
+
+### Tanpa _selector_
+
+Untuk _Service_ _"headless"_ tanpa  _selector_, kontroler _Endpoints_ 
+tidak akan membuat _record_ _Enpoints_. Meskipun demikian, 
+sistem DNS tetap melakukan konfigurasi salah satu dari:
+
+  * _record_ CNAME untuk [_ExternalName_](#externalname)-tipe services.
+  * _record_ untuk semua _Endpoints_ yang memiliki nama _Service_ yang sama, untuk 
+    untuk tipe lainnya.
+
+## Mekanisme _publish_ _Service_ - jenis-jenis _Service_
+
+Untuk beberapa bagian dari aplikasi yang kamu miliki (misalnya saja, _frontend_), 
+bisa saja kamu memiliki kebutuhan untuk mengekspos _Service_ yang kamu miliki 
+ke alamat IP eksternal (di luar kluster Kubernetes).
+
+_ServiceTypes_ yang ada pada Kubernetes memungkinkan kamu untuk menentukan 
+jenis _Service_ apakah yang kamu butuhkan. Secara _default_,jenis _Service_ 
+yang diberikan adalah _ClusterIP_.
+
+_Value_ dan perilaku dari tipe _Service_ dijelaskan sebagai berikut:
+
+   * _ClusterIP_: Mengekspos _Service_ ke _range_ alamat IP di dalam kluster. Apabila kamu memilih _value_ ini 
+     _Service_ yang kamu miliki hanya dapat diakses secara internal. tipe ini adalah 
+     _default_ _value_ dari _ServiceType_.
+   * [_NodePort_](#nodeport): Mengekspos _Service_ pada setiap IP *node* pada _port_ statis 
+     atau _port_ yang sama. Sebuah _Service_ _ClusterIP_, yang mana _Service_ _NodePort_ akan di-_route_
+     akan dibuat secara otomatis. Kamu dapat mengakses _Service_ dengan tipe ini,
+     dari luar kluster melalui `<NodeIP>:<NodePort>`.
+   * [_LoadBalancer_](#loadbalancer): Mengekspos _Service_ secara eksternal dengan menggunakan _LoadBalancer_ 
+     yang disediakan oleh penyedia layanan _cloud_. _Service_ dengan tipe _NodePort_ dan _ClusterIP_, 
+     dimana trafik akan di-_route_, akan dibuat secara otomatis.
+   * [_ExternalName_](#externalname): Melakukan pemetaan _Service_ ke konten 
+      dari _field_ _externalName_ (misalnya: _foo.bar.example.com_), dengan cara mengembalikan 
+      catatan _CNAME_ beserta _value_-nya. Tidak ada metode _proxy_ apa pun yang diaktifkan. Mekanisme ini 
+      setidaknya membutuhkan _kube-dns_ versi 1.7.
+
+### Type NodePort {#nodeport}
```

Comment:
fields and flags should all be in backtick, follow the format according https://deploy-preview-14117--kubernetes-io-master-staging.netlify.com/docs/concepts/services-networking/service/#nodeport